### ISSUE TITLE
ruby_table_view.rb is not included in the ruby gem

### ISSUE THREAD

**Issue by BenGood** [paulfitz/daff#54]
With Daff 1.3.6 installed through ruby gems I don't have ruby_table_view.rb which is required by \lib\coopy\coopy.rb.

**Comment by paulfitz** [comment]
Oops, thanks for reporting this. The following test failed for me in 1.3.6, and is now fixed in 1.3.8:
```ruby
require 'daff'
table1 = [
  ['Country','Capital'],
  ['Ireland','Dublin'],
  ['France','Paris'],
  ['Spain','Barcelona']
]
table2 = [
  ['Country','Code','Capital'],
  ['Ireland','ie','Dublin'],
  ['France','fr','Paris'],
  ['Spain','es','Madrid'],
  ['Germany','de','Berlin']
]
alignment = Daff::Coopy.compare_tables(table1,table2).align()
table_diff = []
flags = Daff::CompareFlags.new
highlighter = Daff::TableDiff.new(alignment,flags)
highlighter.hilite(table_diff)
diff2html = Daff::DiffRender.new
diff2html.use_pretty_arrows(false)
diff2html.render(table_diff)
puts diff2html.html
```
Please let me know if you run into any more ruby glitches. Thanks @BenGood!

**Comment by BenGood** [comment]
Thanks @paulfitz you're quick! 

I did run into another ruby glitch though, see https://github.com/paulfitz/haxe/issues/4
